data/hai/hai reveal 4 middle.txt

```diff
@@ -658,16 +689,58 @@ mission "Pirate Troubles [4]"
 
 
 
+# Temporarily keeping all of this in comments in case I need to draw upon similar functionality in the near future.
```

Comment:
The base game is not where we should be storing stuff for later like this. You should keep these in a local file.